pkg/logs/internal/tailers/file/tailer.go

```diff
@@ -214,8 +209,18 @@ func (t *Tailer) Stop() {
 // This is only used on UNIX.
 func (t *Tailer) StopAfterFileRotation() {
 	t.didFileRotate.Store(true)
+	bytesReadAtRotationTime := t.Source().BytesRead.Load()
 	go func() {
 		time.Sleep(t.closeTimeout)
+		if newBytesRead := t.Source().BytesRead.Load() - bytesReadAtRotationTime; newBytesRead > 0 {
+			log.Infof("After rotation close timeout (%ds), an additional %d bytes were read from file %q.", t.closeTimeout, newBytesRead, t.file.Path)
```

Comment:
Just for consistency with other log messages:
```suggestion
			log.Infof("After rotation close timeout (%ds), an additional %d bytes were read from file %q", t.closeTimeout, newBytesRead, t.file.Path)
```